src/consensus.rs

```diff
@@ -400,12 +405,12 @@ impl <L, M> Consensus<L, M> where L: Log, M: StateMachine {
                 self.leader_state.set_match_index(from, follower_latest_log_index);
                 self.advance_commit_index(actions);
             }
-            Ok(append_entries_response::Which::InconsistentPrevEntry(..)) => {
+            Ok(append_entries_response::Which::InconsistentPrevEntry(index)) => {
```

Comment:
can you just rename `index`->`next_index` and save the `let next_index = index;` below?